.github/workflows/main.yml

```diff
@@ -35,12 +47,19 @@ jobs:
           nvm install --lts
 
       - name: Install uv
-        uses: astral-sh/setup-uv@v5
+        uses: astral-sh/setup-uv@v6
 
       - name: Nx Release flow
         env:
           NX_DAEMON: false
         run: |
-          git config --global user.email "dbbs-lab@cerebhub.eu"
+          git config --global user.email "neurocomp_dbbs@unipv.it"
           git config --global user.name "DBBS Laboratory"
-          ./nx release --yes --verbose
\ No newline at end of file
+          ./nx release version --verbose
+          version=`sed -n -E 's/version = "(.*)"/\1/p' packages/bsb/pyproject.toml`
+          sed -i -E 's/"dateModified": "(\-?[0-9]+){3}"/"dateModified": "'$(date +%F)'"/' codemeta.json
+          sed -i -E 's/"version": "(\.?[0-9]+){3}"/"version": "'$version'"/' codemeta.json
+          sed -i -E 's/\/tag(s)?\/v(\.?[0-9]+){3}/\/tag\1\/v'$version'/' codemeta.json
```

Comment:
Replace sed with a ci/devops script that is human readable, preferably python since that's the language our devs are most fluent in.